clients/src/main/java/org/apache/kafka/common/errors/AuthorizationException.java

```diff
@@ -14,8 +14,20 @@
 
 public class AuthorizationException extends ApiException {
 
+    public AuthorizationException() {
```

Comment:
Do we need this and other similar constructors? Throwing exceptions with no message tends to be not as helpful as one would like and it would be nice to not allow it altogether.